m5_forecasting/utils.py

```
@@ -0,0 +1,58 @@
+class DataChecker:
+    def __init__(self, train_df, test_df, original_row_count, filtered_row_count):
+        self.train_df = train_df
+        self.test_df = test_df
+        self.original_row_count = original_row_count
+        self.filtered_row_count = filtered_row_count
+
+    def not_sales_row_drop_summary_check(self):
+        """Returns the original, filtered, and dropped row counts, and the percentage of dropped rows as a message."""
+        dropped_row_count = self.original_row_count - self.filtered_row_count
+        drop_percentage = (dropped_row_count / self.original_row_count) * 100
+        
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)